pkgs/games/zod/default.nix

```diff
@@ -1,46 +1,104 @@
-{ lib, fetchurl, stdenv, unrar, unzip, SDL, SDL_image, SDL_ttf, SDL_mixer
-, libmysqlclient, makeWrapper }:
-
-stdenv.mkDerivation {
-  name = "zod-engine-2011-03-18";
-
-  src = fetchurl {
-    url = "mirror://sourceforge/zod/zod_src-2011-03-18.zip";
-    sha256 = "00ny7a1yfn9zgl7q1ys27qafwc92dzxv07wjxl8nxa4f98al2g4n";
+{ lib
+, fetchzip
+, stdenv
+, SDL
+, SDL_image
+, SDL_ttf
+, SDL_mixer
+, libmysqlclient
+, gnumake
+, wxGTK
+, symlinkJoin
+, runCommandLocal
+, makeWrapper
+, coreutils
+, scalingFactor ? 2
+}:
+let
+  version = "2011-09-06";
```

Comment:
Is this really the latest version? There appears to have been some development until as late as 2018.

Reply:
it is the latest release that can be acquired from the website ... the quite recent commits are in a attached mercurial repository and I will look into bumping the version again when I had time to test them.